test/tests/transfers/multiapp_interpolation_transfer/fromrestrictedsub_master.i

```diff
@@ -0,0 +1,93 @@
+###########################################################
+# This is a test of the Transfer System. This test
+# uses the Multiapp System to solve independent problems
+# related geometrically. Solutions are then interpolated
+# and transferred from a non-aligned domain.
+#
+# @Requirement F7.20
+###########################################################
```

Comment:
Drop this comment, it's not specifically relevant to your new test. Also, we aren't using this old syntax anymore for the requirements.